.azure/OneBranch.Official.yml

```diff
@@ -0,0 +1,101 @@
+################################################################################
+#                                                                              #
+# This pipeline is for validation of official commits. Useful links:           #
+#                                                                              #
+#   https://www.osgwiki.com/wiki/Windows_Undocked_Template                     #
+#   https://aka.ms/obpipelines                                                 #
+#   https://aka.ms/obpipelines/yaml/schema                                     #
+#   https://aka.ms/obpipelines/tasks                                           #
+#   https://aka.ms/onebranchsup                                                #
+#                                                                              #
+################################################################################
+
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+
+# https://aka.ms/obpipelines/triggers
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+    - release/*
+    - msrc/*
+  tags:
+    include:
+    - v*
+
+schedules:
+- cron: "0 0 * * Sun"
+  displayName: Weekly Validation Run
+  branches:
+    include:
+    - main
+  always: true
+
+# Package and ingestion are disabled by default.
+# Manual runs on the main pipeline are required for packaging/ingestion.
+parameters:
+- name: package
+  type: boolean
+  displayName: Create VPack
+  default: false
+- name: ingest
+  type: boolean
+  displayName: Create Ingest PR
+  default: false
+- name: branch
+  type: string
+  displayName: Branch to PR to
+  default: official/rs_onecore_liof1_stack
+- name: title
+  type: string
+  displayName: PR Title
+  default: 'Automated: Ingest win-net-test'
+
+variables:
+  # https://aka.ms/obpipelines/containers
+  WindowsContainerImage: 'onebranch.azurecr.io/windows/ltsc2022/vse2022:latest'
+
+resources:
+  repositories:
+  - repository: templates
+    type: git
+    name: OneBranch.Pipelines/GovernedTemplates
+    ref: refs/heads/main
+  - repository: undocked
+    type: github
+    name: microsoft/undocked
+    ref: refs/heads/main
+    endpoint: Undocked GitHub
+
+extends:
+  # https://aka.ms/obpipelines/templates
+  template: v2/Microsoft.Official.yml@templates
+  parameters:
+    platform:
+      name: 'windows_undocked'
+    globalSdl:
+      # Disable TSA to force build breaks
+      tsa:
+        enabled: false
+    featureFlags:
+      WindowsHostVersion: '1ESWindows2022'
+    stages:
+    - stage: Build
+      jobs:
+      - template: onebranch/v1/build.yml@undocked
+        parameters:
+          sln: fnmp.sln
+          platform: 'x64,arm64'
+          # Package args
+          package: ${{ parameters.package }}
+          name: 'fnmp'
+          description: 'Win Net Test Helpers'
+          owner: 'cnsdev@microsoft.com'
+          # Windows ingestion args
+          ingest: ${{ parameters.ingest }}
+          osPath: 'minio/net/fnmp'
```

Comment:
Where are we going to want to put this in Windows?

Reply:
onecore\net\test\common\fnmp is probably best.

Reply:
How about `onecore/net/test/common/external` since it's going to just be the manifest and other glue to ingest this stuff?

Reply:
Sure, SGTM